packages/python-packages/protocol-stub-generator/protocol/parse_yml.py

```diff
@@ -78,13 +78,14 @@ def parse_yaml(self):
 
         # Create main_view for LLC API View
         main_view = ProtocolClientView.from_yaml(data)
-
-        # Write output to .json
-        j = open(OUTPUT_FILE, "w")
         parsed_json = json.dumps(main_view.to_json(), default=lambda o: o.__dict__)
-        j.write(parsed_json)
 
-        j.close()
+        # Write output to .json
```

Comment:
@LibbaLawrence @kristapratico Can you please review this change? I have puled changes from Libba's PR with some minor changes.